api/nodes/views.py

```diff
@@ -361,6 +361,19 @@ def perform_destroy(self, instance):
             raise PermissionDenied(str(err))
 
 
+class NodeContributorRemoveMixin:
```

Comment:
Don't add this add a Mixin just over write for the classes you want individually.